packages/components/select/src/select.ts

```diff
@@ -52,50 +66,75 @@ export class Select extends FormControlMixin(LitElement) {
   });
 
   #observer?: MutationObserver;
+  #anchor = new AnchorController(this);
 
   #selectId = `sl-select-${nextUniqueId++}`;
 
-  get allOptions(): SelectOption[] {
-    const groups = this.optionGroups?.map(og => Array.from(og.querySelectorAll('sl-select-option'))) || [];
-    return [...(this.options || []), ...groups.flat()];
-  }
-
   /** Element internals. */
   readonly internals = this.attachInternals();
 
-  /** The current tab node selected in the tab group. */
+  /** The current node selected in the list of options. */
   @state() private selectedOption?: SelectOption | null;
 
+  get allOptions(): SelectOption[] {
+    return Array.from(this.querySelectorAll('sl-select-option'));
+  }
+
+  get #renderSelectedOption(): HTMLElement | TemplateResult {
+    if (!this.selectedOption) {
+      return html`&nbsp;`;
+    }
+    return (this.selectedOption.firstChild as HTMLElement).cloneNode(true) as HTMLElement;
+  }
+
+  get #optionContentType(): string {
+    if (!this.selectedOption) {
+      return `string`;
+    }
+    return (this.selectedOption?.firstChild as HTMLElement).nodeType === 1 ? 'element' : 'string';
+  }
+
   override render(): TemplateResult {
     return html`
-      <div
+      <button
+        ?disabled=${this.disabled}
+        class="select-toggle"
         id=${this.#selectId}
+        popovertarget="dialog-${this.#selectId}"
+        role="combobox"
+        aria-haspopup="listbox"
+        aria-controls="dialog-${this.#selectId}"
+        aria-expanded=${isPopoverOpen(this.dialog)}
+        aria-activedescendant=${this.selectedOption || false}
         tabindex="0"
-        class="select-toggle"
-        @click=${this.openSelect}
-        @keydown="${this.#handleOverallKeydown}"
       >
-        <span id="selectedOption" style=${styleMap(this.size || {})}></span>
-
-        <div class="toggle-icon" aria-label="Choose" role="button">🔽</div>
-        <!-- to be replaced by <sl-icon></sl-icon> -->
-      </div>
-      <sl-select-overlay
-        @keydown=${this.#handleOverlayKeydown}
+        <span aria-hidden=${!this.selectedOption} contenttype=${this.#optionContentType}
+          >${this.#renderSelectedOption}</span
+        >
+        <sl-icon name="chevron-down"></sl-icon>
+      </button>
+      <dialog
+        @beforetoggle=${this.#setPopoverWidth}
         @click=${this.#handleOptionChange}
+        @keydown="${this.#handleOverlayKeydown}"
+        @toggle=${this.#handleOptionFocus}
+        id="dialog-${this.#selectId}"
+        anchor=${this.#selectId}
+        popover
+        ${anchor({ position: 'bottom' })}
         aria-labelledby=${this.#selectId}
-        style="--max-overlay-height:${this.maxOverlayHeight || 'unset'}"
+        role="listbox"
       >
         <slot @slotchange=${this.#handleOptionsSlotChange}></slot>
-      </sl-select-overlay>
+      </dialog>
       ${this.#validation.render()}
```

Comment:
I don't think we do; there was a discussion about that right? that you can't see the hint anymore when you open the dropdown. And it shouldn't be needed, since the fact that you have a list of option is already guiding the user on what needs to be filled in.
@RoaldBoerema do you agree with that / did i remember it correctly?